ui/eureka/eureka.js

```diff
@@ -18,10 +18,56 @@ let Options = {
       'Eureka Hydatos': '丰水之地',
     },
     ko: {
-      'Eureka Anemos': '아네모스 지대',
-      'Eureka Pagos': '파고스 지대',
-      'Eureka Pyros': '피로스 지대',
-      'Eureka Hydatos': '히다토스 지대',
+      'Eureka Anemos': '아네모스편',
+      'Eureka Pagos': '파고스편',
+      'Eureka Pyros': '피로스편',
+      'Eureka Hydatos': '히다토스편',
+    },
+  },
+  timeStrings: {
+    weatherFor: {
+      en: (nowMs, stopTime) => {
```

Comment:
Can you include the weather in here too, in case some other language wants the time before the "for" part? Sorry for all the requests, I just think that localized text should be entirely localized and not partially constructed by appending strings.

Reply:
I don't know what you're saying about.
Only I can think of is what I've already written below `weatherIn`.

Reply:
for "entirely localized and not partially consturcted", I can only think of `minute` part, which only changes word, not word's location.

Reply:
Sorry for the confusion.  What I would like to avoid is partial/incomplete localized strings being appended to other partial strings to create a final result.  Appending strings together implicitly creates an ordering for grammar that doesn't apply consistently to all languages.  In this case, you have `weatherFor` which returns a string like `for 3m` (a partial string) and is appended to some weather emoji (the rest of the string).  What happens if some other language wants to put the weather emoji after the time and not before? Then this needs to be rewritten.

I think it would be better to have `weatherFor` return the complete string, e.g. `☃️ for 3m`.  That's why I was trying to suggest that `weatherFor` look like `en: (weather, nowMs, stopTime) => {` instead.

Sorry for all the changes.  Mostly I wanted to communicate this, as it seems likely it will come up again for other cases.

Reply:
ah, you meant some condition with weather icon. I was actually thinking about separating weather icon part and text part to align text.
because weather icon width is different themselves.
![K-002](https://user-images.githubusercontent.com/4695371/79702127-6d382200-82dd-11ea-9745-450e02a575f8.png)
Those were something I was thinking about this.

The real response with your comment.
I don't think those icons will move front or back, because people don't read icon, they just see the images.

For Korean, it's better to say
"3분 동안 눈"
first, time part
next, means "for"
lastly, means "snow"

but as this is an icon, it does not follow grammar.
It's like... hmmm
just indicating. you might see images for order, not thinking about time thing.
I cannot explain accurately because of my lack of English.

Reply:
maybe for right-align written languages can be better with back side icons.
but hmmm.... I think separating text part and icon part, then fixing icon on the left is better.

Reply:
Oh, I was thinking about right-to-left written languages, but they are not written backward.
like not "English", "hsilgnE" with right sided.
So, icons with in front is better.

Reply:
Separating sounds nice and would align better.  However, how do you differentiate between "current weather for X minutes" vs "new weather in X minutes"?

Reply:
And yeah, in English, the icon is intended to be part of the sentence.  It's "snow for 20 minutes" etc.

Reply:
I was thinking about making separate div.
I didn't think deeply though.

Reply:
![K-002](https://user-images.githubusercontent.com/4695371/79705645-5b607a00-82f1-11ea-8d26-2263e2d8361e.png)

Reply:
Ah, separate div seems good.  Maybe the text can be the same in English, and that would be fine.

Reply:
Ooh, I like this a lot, thanks!